pypeit/reduce.py

```diff
@@ -84,14 +84,16 @@ class Reduce(object):
     __metaclass__ = ABCMeta
 
     def __init__(self, spectrograph, tslits_dict, mask, ir_redux=False, det=1, objtype='science', binning=None, setup=None,
-                 par=None, frame_par=None):
+                 par=None, maskslits=None):
 
         # Setup the parameters sets for this object. NOTE: This uses objtype, not frametype!
         self.objtype = objtype
-        self.par = pypeitpar.ScienceImagePar() if par is None else par
-        self.frame_par = pypeitpar.FrameGroupPar(objtype) if frame_par is None else frame_par
-        self.proc_par = self.frame_par['process']
-
+        self.par = spectrograph.default_pypeit_par() if par is None else par
```

Comment:
Make `par` a required argument and remove this.

Reply:
Done.